atlas/packages/components/src/components/Dropdown/Dropdown.tsx

```diff
@@ -15,23 +13,23 @@ type DropdownProps = {
 	label: string;
 	helper?: string;
 	value?: string;
-	options: Array<DropdownOption>;
+	options: DropdownOption[];
 	onChange?: (option: DropdownOption) => void;
 } & DropdownStyleProps;
 
 export default function Dropdown({
 	label,
 	helper = "",
 	value = "",
-	options,
+	options = [],
 	disabled = false,
 	onChange = () => {},
 	...styleProps
 }: DropdownProps) {
 	const inputRef = useRef(null);
 	const [isActive, setIsActive] = useState(!!value);
 	const [inputTextValue, setInputTextValue] = useState(
-		!!value ? options.find(option => option.value === value).text : ""
+		!!value ? options.find(({ value: optionValue = NaN }) => optionValue === value)?.text : ""
```

Comment:
Why do we use `NaN` as a default value here?

Reply:
Was nonsense, changed it to an empty string instead.